static/src/javascripts/bootstraps/enhanced/accessibility.js

```diff
@@ -82,13 +84,13 @@ define([
 
         React.render(
             React.createElement(Accessibility),
-            document.getElementById(module.DOM_ID),
+            document.getElementById(DOM_ID),
```

Comment:
I prefer not having to rely on other libraries and their magic to write tests.

```js
var module = {
   DOM_ID: 'js-accessibility-preferences',
   init: init
};

function init () {}

return module;
```

^ not a mutation and should pass lint